discord-bot-demo/backend/supabase/functions/interactions/user.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  APIChatInputApplicationCommandInteraction,
+  APIEmbed,
+  Snowflake,
+} from "https://deno.land/x/discord_api_types@0.36.1/v10.ts";
+import { stripIndents } from "https://deno.land/x/deno_tags@1.8.0/tags/stripIndents.ts";
+import { getDiscordConnection } from "../util/truffle/index.ts";
+import { createResponse } from "../util/respond.ts";
+
+const orgId = Deno.env.get("TRUFFLE_ORG_ID")!;
```

Comment:
We can probably go ahead and add the code now to grab the orgId from the discordGuildId, we can run a script on `truffle-dev-early-access` to add the Connection in the interim until we build out the admin side to add Discord guild's